src/Controls/samples/Controls.Sample.UITests/Controls.Sample.UITests.csproj

```diff
@@ -45,6 +45,30 @@
     <MauiSplashScreen Include="Resources\Splash\splash.svg" Color="#FFFFFF" BaseSize="168,208" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
The IDE adds this by accident

Can you remove all these?

Reply:
Done